bpf-gpl/nat.h

```diff
@@ -428,7 +428,6 @@ static CALI_BPF_INLINE int vxlan_attempt_decap(struct cali_tc_ctx *ctx) {
 	return -2;
 
 deny:
-	ctx->fwd.res = TC_ACT_SHOT;
```

Comment:
This is also done in tc program for the return value of -1. Removed it to make it generic even for XDP programs.

Reply:
It is not needed by XDP. I was trying to make some functions generic. I'll revert this and raise it in another PR.